tox.ini

```diff
@@ -122,7 +122,8 @@ envlist =
     py3{6,7,8,9}-test-instrumentation-grpc
 
     ; opentelemetry-instrumentation-sqlalchemy
-    py3{6,7,8,9}-test-instrumentation-sqlalchemy{11,14}
+    py3{6,7}-test-instrumentation-sqlalchemy{11}
```

Comment:
Can you add a comment explaining why we do this?

Reply:
https://github.com/open-telemetry/opentelemetry-python-contrib/pull/531/files#r696214503

Will add a comment to the file.